cmd/baton-calendly/main.go

```diff
@@ -37,12 +37,12 @@ func main() {
 	}
 }
 
-func prepareAuth(cfg *config) uhttp.AuthCredentials {
+func prepareAuth(cfg *config) *uhttp.BearerAuth {
 	if cfg.Token != "" {
 		return uhttp.NewBearerAuth(cfg.Token)
 	}
 
-	return &uhttp.NoAuth{}
+	return &uhttp.BearerAuth{}
```

Comment:
Doesn't this cause us to tell the http client to send a header with an empty auth token instead of no auth header? One solution is to pass cfg.Token into connector.New(), and have the New() function create a BearerAuth client or NoAuth client based on whether the token is empty or not.

Reply:
Fixed